### PR TITLE
Rewrite content type if image

### PR DESCRIPTION
**Pull request by sihu**
after deploying this, reset the `ENV` and change https://console.aws.amazon.com/cloudfront/home?region=us-east-1#distribution-settings:E22LXDA45QLQRU from /o/ to the custom server.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -21,16 +21,31 @@ def generate(r):
...
+        print('serving with headers ' + str(headers))
+
     return Response(generate(r), headers=headers, status=r.status_code)
 
 @app.route('/healthcheck')
 def healthcheck():
     return 'WORKING'
 
+@app.route('/o/<path:uri>')
+def serve_image_replacing_images_mime_type(uri):
```
Comment (sihu) [inline server.py:44]: after deploying, the `BACKEND_ASSET_PATH` has to be changed to `d2ssnmyozoo4xu.cloudfront.net` so there is no endless loop

**Review by sihu — COMMENTED** [review]
comment

File: server.py [hunk 2]
```diff
@@ -21,16 +21,31 @@ def generate(r):
     for chunk in r.iter_content(chunk_size):
         yield chunk
 
+def is_image(out_path):
+    return out_path.endswith('.jpg') or out_path.endswith('.jpeg') or out_path.endswith('.png') or out_path.endswith('.gif')
```
Comment (macav) [inline server.py]: how about
```python
ext = out_path.split('.')[-1]
return ext in ['jpg', 'jpeg', 'png', 'gif']
```

it doesn't change a lot, but maybe it's a bit more elegant :)

File: server.py [hunk 3]
```diff
@@ -21,16 +21,31 @@ def generate(r):
...
+
 def fetch_image(out_path):
     r = requests.get(out_path, stream=True, params=request.args)
     headers = dict(r.headers)
 
+    if headers['Content-Type']  == "binary/octet-stream" and is_image(out_path):
+            headers['Content-Type'] = "image/jpeg"
+    if app.debug:
+        print('serving with headers ' + str(headers))
```
Comment (macav) [inline server.py:35]: in general there should be some kind of logger where you can output stuff on debug level, but i don't know how it's done normally within these project

**Review by macav — APPROVED** [review]
fine for me, i just left some comments, but just suggestions

File: server.py (hunk 3 above)
Comment (sihu) [reply server.py:35]: i copied this from other parts in this file. i'd also prefer a logger but not sure if it's worth to od it

**Review by sihu — COMMENTED** [review]
refactored the is_image

**Pull request merged**